wger/utils/helpers.py

```diff
@@ -126,6 +127,30 @@ def make_token(user):
     return uid, token
 
 
+def get_exercise_as_json(exercise, image, thumbnail):
+    exercise_json = {
+        'value': exercise.name,
```

Comment:
This dictionary could be in that helper function I talked about to reduce the number of lines as its a nice practice